pkg/action/install.go

```diff
@@ -69,10 +69,8 @@ type Install struct {
 
 	ChartPathOptions
 
-	ClientOnly      bool
```

Comment:
We can't remove public properties on structs because it would be a breaking change to the Go clients API. We can deprecate things but not remove them until there is a major version change for Helm.

Reply:
Yes, totally agree, missed that moment, reverted my changes, and fixed in an easy way.  But I think we should start deprecating some of the fields because they duplicate the same logic so you can set ClientOnly=true, or DryRun="client"(or "true"), some code checks ClientOnly, some code checks DryRun